OWNERS

```diff
@@ -9,4 +9,6 @@ approvers:
   - jwforres
   - knobunc
   - sjenning
-  - soltysh
\ No newline at end of file
+  - soltysh
+  - bparees
+  - sdodson
```

Comment:
@sdodson fyi

Reply:
I'd be totally fine if you omit me, at least for the next 2 months.

Reply:
heh, fair enough. updated.